### PR TITLE
Revert lockfile to version 1, update clean command.

### PR DESCRIPTION
**Pull request by sblaurock**
#85 will force us to be using the current Node LTS before versioning this package, which uses an npm version with lockfile version 1. This is preferable for now until we upgrade to the next LTS.

All tests pass.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Constructor.io JavaScript client",
   "main": "lib/constructorio.js",
   "scripts": {
-    "clean": "rm -rf node_modules",
+    "clean": "sudo rm -rf node_modules package-lock.json && npm install",
```
Comment (jjl014) [inline package.json:7]: Should we leave out `npm install` here to be consistent with the other repos?

**Review by jjl014 — COMMENTED** [review]
👍

**Comment by sblaurock** [comment]
@jjl014 updated - great catch

**Review by jjl014 — APPROVED** [review]
🚢

**Pull request merged**